Sources/swift-format/Subcommands/LintFormatOptions.swift

```diff
@@ -94,6 +105,10 @@ struct LintFormatOptions: ParsableArguments {
       throw ValidationError("'--assume-filename' is only valid when reading from stdin")
     }
 
+    if offsets?.isEmpty == false && paths.count > 1 {
```

Comment:
It might be me but I always need to think extra hard with the optional to boolean comparisons and prefer coalescing the `nil` value.

```suggestion
    if !(offsets?.isEmpty ?? true) && paths.count > 1 {
```

Reply:
I find the extra `!(...)` to be pretty obtuse. The idiom I'm using seems far more common (and readable) to me. 🙃

Reply:
OK, then it’s just me 😉

Reply:
FWIW, I also find the `== false` version easier to read. 🙂

Reply:
And this goes away with `offsets` no longer being optional. 😀